app/serializers/api/application_serializer.rb

```diff
@@ -32,7 +32,12 @@ class AttributesSerializer < Blueprinter::Base
       field(:itt_provider) { |a| a.itt_provider&.legal_name }
       field(:lead_mentor)
       field(:funded_place)
-      field(:created_at)
+      field(:created_at) do |a|
+        [
+          a.accepted_at,
```

Comment:
Is it worth sticking this logic in `Application` as something like `latest_created_or_accepted_at` so we're not duplicating it in the serializers and we can add dedicated tests?

Reply:
I think in the application serialiser we should stick to the created at, only in the participant serialisers we change this value?